src/shared/components/cancerSummary/SummaryBarGraph.tsx

```diff
@@ -0,0 +1,215 @@
+import * as React from "react";
+import * as _ from 'lodash';
+import {computed, observable} from "mobx";
+import { ChartTooltipItem } from '@types/chart.js';
+import Chart, {ChartLegendItem} from 'chart.js';
+import {IBarGraphData} from './CancerSummary';
+import {observer} from "mobx-react";
+import './styles.scss';
+
+interface ISummaryBarGraphProps {
+    data: IBarGraphData;
+    yAxis: 'alt-freq' | 'abs-count';
+    xAxis: 'y-axis' | 'can-types';
+    legend: boolean;
+    setPngAnchor:any;
+    setPdfAnchor:any;
+}
+
+interface IChartLegendItem extends ChartLegendItem {
+    datasetIndex: number;
+}
+
+@observer
+export default class SummaryBarGraph extends React.Component<ISummaryBarGraphProps, {}> {
+
+    private chartTarget:HTMLCanvasElement;
+
+    @observable private chart:any;
```

Comment:
if you dont want every member of `chart` to become observable but you just want to be able to react to when `SummaryBarGraph::chart` is changed, you should use `@observable.ref`